Jenkinsfile

```diff
@@ -15,6 +17,9 @@ pipeline {
     disableResume()
     durabilityHint('PERFORMANCE_OPTIMIZED')
   }
+  triggers {
+    issueCommentTrigger('.*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tav\\W+tests(?:\\W+please)?.*')
```

Comment:
"tav" is not a thing for the Go agent, I guess this was copied from the Node.js agent? So just remove the tav bit, and then LGTM.

Reply:
yep C&P

Reply:
Thanks, merge at will